src/scripts/web3-interface.ts

```diff
@@ -27,34 +28,42 @@ export interface Web3Interface {
   viewStateFacet: ViewStateFacet;
 }
 
+// TODO -- consider instantiating this once per session, or memo-izing with cache timeout
 export const getWeb3Interface = async (isTest?: boolean): Promise<Web3Interface> => {
   try {
-    const rpcProvider = new ethers.providers.JsonRpcProvider(process.env.PROVIDER_URL);
+    const networkConfig = getNetworkConfigByChainId(process.env.CHAIN_ID || localChainId);
```

Comment:
was having build issues when setting this to `process.env.CHAIN_ID` -- this will still load the correct chain ID when running the `cli`